incident_test.go

```diff
@@ -155,6 +155,44 @@ func TestIncident_Manage_priority(t *testing.T) {
 	testEqual(t, want, res)
 }
 
+func TestIncident_Manage_title(t *testing.T) {
+	setup()
+	defer teardown()
+
+	mux.HandleFunc("/incidents", func(w http.ResponseWriter, r *http.Request) {
```

Comment:
Yeah, I get that. I've a desire to rework the tests to more clearly assert that we're matching the API's expectations, so I was trying to avoid having more places where I need to worry about testing for that in the future. To your point, there are no patterns of how to really do that well yet.